ray-operator/config/samples/ray-cluster.profiling.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: ray.io/v1alpha1
+kind: RayCluster
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+    # An unique identifier for the head node and workers of this cluster.
+  name: raycluster-profiling
+spec:
+  rayVersion: '2.5.0' # should match the Ray version in the image of the containers
+  # Ray head pod template
+  headGroupSpec:
+    rayStartParams:
+      dashboard-host: '0.0.0.0'
+    #pod template
+    template:
+      spec:
+        containers:
+        - name: ray-head
+          image: rayproject/ray:2.5.0
+          resources:
+            limits:
+              cpu: 1
+              memory: 2Gi
+            requests:
+              cpu: 500m
+              memory: 2Gi
+          # `py-spy` is a sampling profiler that requires `SYS_PTRACE` to read process memory effectively.
+          # Once enabled, you can profile Ray worker processes through the Ray dashboard.
+          # For more details, refer to:
+          # https://docs.ray.io/en/latest/ray-observability/user-guides/debug-apps/optimize-performance.html#python-cpu-profiling-in-the-dashboard
+          securityContext:
+            capabilities:
+              add:
+              - SYS_PTRACE
+          ports:
+          - containerPort: 6379
+            name: gcs-server
+          - containerPort: 8265 # Ray dashboard
+            name: dashboard
+          - containerPort: 10001
+            name: client
+          volumeMounts:
+            - mountPath: /home/ray/samples
+              name: ray-example-configmap
+        volumes:
+          - name: ray-example-configmap
+            configMap:
+              name: ray-example
+              items:
+                - key: long_running_task.py
+                  path: long_running_task.py
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ray-example
+data:
+  long_running_task.py: |
+    import ray
+    import time
+
+    @ray.remote
+    def long_running_task():
+      while True:
```

Comment:
Make the task run forever. This buys time for users to view the Stack Trace and CPU Flame Graph via the Ray dashboard. We cannot use 'sleep' here, as using sleep will cause the process to be idle, leading to the "No stack counts found" in the CPU Flame Graph. See https://github.com/benfred/py-spy/issues/321#issuecomment-731848950 for more details.

Reply:
Could you add this to the doc?